src/main/java/BowlingGameKata/Game.java

```diff
@@ -0,0 +1,107 @@
+package BowlingGameKata;
+
+import java.util.List;
+
+public class Game {
+
+
+    private int score = 0;
+    private boolean isSpare = false;
+    private boolean isStrike = false;
+
+
+    void roll(int pinsDropped){
```

Comment:
there are several package private methods here that are tested. It means we lack "the game" itself. Entry point should be some method e.g. play/roll/whatever 😄 and all the calculation should be done under the hood.
right now I cannot play because such method is not available.

Reply:
additionaly, when you are doing TDD you suppose to be focused on one functionality (one entry point/method/function) - thanks to that, you know the requirements regarding the functionality are met. In your solution, there are pieces but I have no clue if they works together